src/styles/main.scss

```diff
@@ -0,0 +1,6 @@
+@import './utils/reset';
+@import './utils/normalize';
+@import './utils/variables';
+@import './utils/colors';
+@import './utils/mixins';
+@import './blocks/calendar';
```

Comment:
Ensure that all imported files (`reset`, `normalize`, `variables`, `colors`, `mixins`, and `calendar`) exist in the specified paths. These files should contain the necessary definitions for variables and mixins used in your SCSS files.